xml/System/String.xml

```diff
@@ -14018,13 +14043,13 @@ A string is empty if it  is explicitly assigned an empty string ("") or <xref:Sy
   
  White-space characters are defined by the Unicode standard. The <xref:System.String.Trim> method removes any leading and trailing characters that produce a return value of `true` when they are passed to the <xref:System.Char.IsWhiteSpace%2A?displayProperty=nameWithType> method.  
   
-   
+[!INCLUDE[interactive-note](~/includes/csharp-interactive-note-some.md)]  
   
 ## Examples  
  The following example uses the <xref:System.String.Trim?displayProperty=nameWithType> method to remove any extra white space from strings entered by the user before concatenating them.  
   
  [!code-cpp[System.String.Trim#2](~/samples/snippets/cpp/VS_Snippets_CLR_System/system.String.Trim/cpp/trim2.cpp#2)]
- [!code-csharp[System.String.Trim#2](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.String.Trim/cs/Trim2.cs#2)]
+ [!code-csharp-interactive[System.String.Trim#2](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.String.Trim/cs/Trim2.cs#2)]
```

Comment:
this doesn't seem to work well with try .net because it needs input and it doesn't seem to behave very well. I'd undo the change to this snippet (in the samples too)

Reply:
Done, and snippet fixed in https://github.com/dotnet/samples/pull/1165